msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Resource-Specific Consent (RSC) permission preapproval provides admins with detailed control over the RSC permissions that an app can request during installation. The app only receives RSC permissions at the installation time. However, using preapproval policies, admins can predetermine the maximum permissions that an app can request from the end user and secure consent during installation.
```

Comment:
Please replace the first para with this:

Resource-Specific Consent (RSC) permissions preapproval provides admins with granular control over the RSC permissions an app can request upon installation. Today, RSC permissions are granted to an app at the time the app gets installed. Through the use of pre-approval policies, admins can declare ahead of time the max permissions an app can request from the end user and be consented to during installation time.

Reply:
@AkJo ,  as per our documentation standards, we avoid using terms like “Today” or “Currently” and refrain from mentioning future updates to avoid the sharing of outdated information should changes occur.